composer.json

```diff
@@ -175,6 +175,9 @@
             },
             "drupal/geocoder": {
                 "3224364 - Replace deprecated boolean return in uksort() for PHP 8 compatibility.": "https://www.drupal.org/files/issues/2021-07-19/3224364-2.patch"
+            },
+            "drupal/search_api": {
+                "Resolves call to member function preExecute() on null error on site install": "https://www.drupal.org/files/issues/2021-09-20/search_api_condition_cacheable_metadata.patch"
```

Comment:
@panshulK - We should also add this patch in acquia_cms_search module as well so that end user using custom profile can get this patch.

Reply:
Done @chandan-singh7929